pac4j-saml/src/main/java/org/pac4j/saml/metadata/SAML2ServiceProviderMetadataResolver.java

```diff
@@ -44,12 +44,14 @@
     private final boolean forceSpMetadataGeneration;
     private boolean authnRequestSigned;
     private boolean wantsAssertionsSigned;
+    private final String nameIdPolicyFormat;
```

Comment:
In fact, we don't use that property anywhere else, it is just passed to the metadata generator. I think we could remove it.